### PR TITLE
[DOC] - Added description of units assumed in shuffle functions

### PR DESCRIPTION
**Pull request by maestapereira**
When working through some of the functions, we noticed and inconsistency in spike units. To help address this, this PR adds a specification that spike times should be in ms.

What's added:
* adds milliseconds to description of spikes in functions `shuffle_circular` and `shuffle_bins`.

What's next:
* we still need to update any other functions that assume milliseconds as the unit of spike.

### CODE REVIEW

File: spiketools/stats/shuffle.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def shuffle_circular(spikes, shuffle_min=20000, n_shuffles=1000, random_state=No
     Parameters
     ----------
     spikes : 1d array
-        Spike times.
+        Spike times, in milliseconds.
     shuffle_min : int
         The minimum amount to rotate date, in terms of units of the spike train.
```
Comment (TomDonoghue) [inline spiketools/stats/shuffle.py:194]: I just noticed there is a type here ('date' which should be 'data'). I know this typo doesn't come from your changes here, but could you fix it in this PR?

File: spiketools/stats/shuffle.py [hunk 2]
```diff
@@ -199,6 +199,10 @@ def shuffle_circular(spikes, shuffle_min=20000, n_shuffles=1000, random_state=No
     -------
     shuffled_spikes : 2d array
         Shuffled spike times.
+		
+    Notes
+    -----
+    The input shuffle_min should always be less than the number of spikes (of input spikes).
```
Comment (TomDonoghue) [inline spiketools/stats/shuffle.py:205]: I'm not sure I totally follow this comment. I think maybe the value of `shuffle_min` should be less than the duration of the spike train (for example, if there are 100 spikes covering time range 0 to 10000 ms, should shuffle_min be less than 10000? I don't think it should be that the value of shuffle_min needs to be less than 100 though). Could you rephrase this?

**Review by TomDonoghue — CHANGES_REQUESTED** [review]

**Comment by TomDonoghue** [comment]
This looks great! I've added a couple comments, one to add another quick fix here, and another suggesting some clarifications to the note. Let me know if anything is unclear!

By the way - to add / update changes to this PR, you can just push commits to the relevant branch, everything in the PR will automatically update accordingly. Tag me back in when it's ready to look at again!

File: spiketools/stats/shuffle.py (hunk 2 above)
Comment (maestapereira) [reply spiketools/stats/shuffle.py]: You're right, I did not describe it correctly! I changed the wording, please let me know if this is clearer, or if I should work a little more on it.

File: spiketools/stats/shuffle.py (hunk 1 above)
Comment (maestapereira) [reply spiketools/stats/shuffle.py]: Good catch! Just fixed it.
